schemas/session-replay/_common-shape-wireframe-schema.json

```diff
@@ -16,6 +16,7 @@
           "properties": {
             "backgroundColor": {
               "type": "string",
+              "pattern": "^[#][a-f0-9]{2}[a-f0-9]{2}[a-f0-9]{2}([a-f0-9]{2})?",
```

Comment:
**suggestion:** simplify and add `$` at the end: `^#[a-f0-9]{6}([a-f0-9]{2})?$`

https://regex101.com/r/61Jo1w/1